packages/components/src/markdown/index.tsx

```diff
@@ -1,22 +1,34 @@
-import PropTypes from 'prop-types';
 import clsx from 'clsx';
 import { marked } from 'marked';
 import styles from './index.module.scss';
 
 import { sanitizeText } from './utils';
-import { useClickHandler } from './hooks';
+import { ClickHandler, useClickHandler } from './hooks';
+import { ComponentPropsWithoutRef, PropsWithChildren } from 'react';
 
-const options = { gfm: false };
+const options = { gfm: false } as const;
+
+type RemainingHTMLAttributes = Omit<ComponentPropsWithoutRef<'span'>,
+'className' | 'onClick'
```

Comment:
are we omiting them here to be more explicit that className is allowed as a Prop or why is it?

Reply:
type includes all props a span would have **but className and onClick**

Reply:
I see, but why?

Reply:
because we're setting them in line 38

Reply:
https://github.com/goodhood-eu/goodhood/blob/32edd4448837dec08ae252bf2922f9175c23d137/packages/components/src/markdown/index.tsx#L38